functions/graphql/schema/auth.js

```diff
@@ -0,0 +1,23 @@
+const { gql } = require("apollo-server-express");
+
+// We define a schema that encompasses all of the types
+// necessary for the functionality in this file.
+module.exports.schema = gql`
+  # We can extend other graphQL types using the "extend" keyword.
+  extend type Mutation {
+    login(email: String!, password: String!): User
+    signUp(email: String!, password: String!): User
+    logout: User
+  }
+`;
+
+// We define all of the resolvers necessary for
+// the functionality in this file. These will be
+// deep merged with the other resolvers.
+module.exports.resolver = {
+  Mutation: {
+    login: (_, { email, password }, context) => {},
```

Comment:
What do the underscores mean?

Reply:
Underscores mean the argument is not used in the function, so it doesn't need a proper variable name.